test/performance/benchmarks/broker-pubsub/cluster.yaml

```diff
@@ -16,6 +16,6 @@
 
 GKECluster:
   location: "us-west1"
-  nodeCount: 3
```

Comment:
Can we keep it the same as eventing? I think we use 3 there? 
Just to be able to do more of an apple to apple comparison, and have an idea of how we compare to the IMC...

Reply:
I just changed the cluster size to 1 in eventing, see https://github.com/knative/eventing/pull/2174, since I found the benchmarks had very low resource utilization. The node count here is also derived from my observation when the benchmark were run.

I'm not sure if cluster size is one factor that we need to consider when we compare it with IMC, since obviously pubsub channel will need more resources (it requires Serving, Istio, and some other pubsub related resources). IMO as long as the cluster is big enough for running the benchmarks and getting metrics, it would be fine.

Reply:
OK, I can live with this! Thanks Chi